pvr.vuplus/resources/language/resource.language.de_de/strings.po

```diff
@@ -116,6 +116,10 @@ msgctxt "#30028"
 msgid "Use Secure HTTP (https)"
 msgstr "Sicheres HTTP (https) verwenden"
 
+msgctxt "#30030"
```

Comment:
Please drop the German translation completely, these are done using Transifex. When you add new strings you only need to update `resource.language.en_gb`.